alg/viewshed/viewshed.h

```diff
@@ -83,10 +81,10 @@ class Viewshed
 
   private:
     Options oOpts;
-    Window oOutExtent;
-    Window oCurExtent;
-    DatasetPtr poDstDS;
-    GDALRasterBand *pSrcBand;
+    Window oOutExtent{};
```

Comment:
I'm curious: what changes here? Isn't the default ctor called in both cases?

Reply:
The help please -Weffc++ given that the constructor no longer explicitly initialize those members.